### DISCUSSION TITLE
How do I detect changes of accumatlive cost?

### DISCUSSION THREAD

**Discussion by JRascagneres** [BottlecapDave/HomeAssistant-OctopusEnergy#649] [Q&A]
I think this 'broke' a while ago and I didn't really notice but...

I have an automation detecting a change in the accumulative costs: this is the 'normal', the export and the override. This seems to trigger every 30 minutes now. Anything I can do?

**Comment by BottlecapDave** [comment]
I'm assuming every 30 minutes is too much? The automation probably needs to omit looking at attributes. This can be done by either setting this behaviour in the UI or if you use yaml you need `for: null` in your trigger.

**Reply by JRascagneres** [reply]
It might be okay actually...
My use case is basically calculating difference between my 'normal' cost and the override cost to calculate my saving. Previously if I didn't do this check and wait for them both to update one could 'lag' behind and I'd end up with the stat looking a bit weird for a minute or two but maybe this won't happen anymore?
